### ISSUE TITLE
update 2.3.1 to 2.4.1

### ISSUE THREAD

**Issue by defaliz** [billz/raspap-webgui#617]
**Before submitting an issue**
Please read this first https://github.com/billz/raspap-webgui/wiki/Reporting-issues

* [ ] This is a bug report
* [ ] I searched existing issues before opening this one
* [ ] I checked the FAQ before creating this issue
* [ ] I have read and understand the issue reporting guidelines 

**Describe the bug**

**Your environment**
* Raspberry Pi hardware (examples: Pi 3 Model B+, Pi Zero W)  
* Raspbian version (examples: Buster Lite, Buster Desktop)
* Followed the project prerequisites? (Y/N)
* Checked the project FAQ? (Y/N)
* RaspAP Quick Install or Manual setup?
* Using default configuration? (Y/N)
* Simultaneous AP and managed mode? (Y/N)
* Onboard wireless chipset or external adapter? 
* Other software or services running with RaspAP?

**Steps to reproduce**
Tell us how to reproduce this issue. Provide as much detailed information as possible.

**Expected behavior**

**Actual behavior**
Tell us what you observed instead.

**Screenshots**

**Additional context**

**Comment by defaliz** [comment]
hi, how upgrade v2.3.1 to v2.4.1 ? and keeping my configuration ?
regards

**Comment by GiannisDorokidis** [comment]
https://github.com/billz/raspap-webgui/wiki/FAQs#what-are-the-steps-to-upgrade-raspap

**Comment by defaliz** [comment]
great !
thank you very much..

**Comment by billz** [comment]
Version 2.4 includes ad blocking, which needs some [extra steps to be performed](https://github.com/billz/raspap-webgui/wiki/Manual-installation) that aren't covered by the FAQ.

An `--upgrade` option for the installer would simplify things.

**Comment by billz** [comment]
Quick installer wiki page + FAQ are updated. 
tl;dr: run `curl -sL https://install.raspap.com | bash -s -- --upgrade`

@defaliz thanks for reporting
